pnpm-lock.yaml

```diff
@@ -1705,6 +1719,7 @@ packages:
   /@babel/helper-validator-option/7.16.7:
     resolution: {integrity: sha512-TRtenOuRUVo9oIQGPC5G9DgK4743cdxvtOw0weQNpZXaS16SCBi5MNjZF8vba3ETURjZpTbVn7Vvcf2eAwFozQ==}
     engines: {node: '>=6.9.0'}
+    dev: true
```

Comment:
saw this back and forth, kinda annoying

Reply:
Yeah, me too. Not sure what that was all about but I didn't dive into it since it's auto generated by pnpm

Reply:
Just googled it and it seems `dev: true` indicates it's a dev dependency. Not sure why it would be gone sometimes, but at least it SHOULD be there.
https://stackoverflow.com/questions/49809490/what-is-dev-true-in-package-lock-json-for

Reply:
well this is pnpm so it may have different meaning, not sure about it.

Reply:
OK, I've created a [ticket](https://linear.app/silverhand/issue/LOG-2857/investigate-the-random-dev-true-in-pnpm-lockfile) and let's see if we can find anything when we have time